src/index.html

```diff
@@ -11,6 +11,52 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
+      action="/"
+      class="form"
+      method="get"
+      data-qa="big"
+    >
+
+      <img
```

Comment:
No need to add blank lines between child and parend elements
Check all such cases